store/localstore/local_region.go

```diff
@@ -83,6 +207,30 @@ func (rs *localRegion) Handle(req *regionRequest) (*regionResponse, error) {
 			ctx.whereColumns = make(map[int64]*tipb.ColumnInfo)
 			collectColumnsInExpr(sel.Where, ctx, ctx.whereColumns)
 		}
+		if len(sel.OrderBy) > 0 {
+			if sel.OrderBy[0].Expr == nil {
```

Comment:
If orderby[0].expr is nil and sel.orderby[0].desc = false, it means this request is desc scanned.